File: GerenciaMusic360.Entities/MarketingPlan.cs
using System;
using System.Collections.Generic;

namespace GerenciaMusic360.Entities
{
    public partial class MarketingPlan
    {
        public int Id { get; set; }
        public int MarketingId { get; set; }
        public int TaskDocumentDetailId { get; set; }
        public short Position { get; set; }
        public string Notes { get; set; }
        public DateTime? EstimatedDateVerification { get; set; }
        public bool Required { get; set; }
        public bool Status { get; set; }
        public string Name { get; set; }
        public string EstimatedDateVerificationString { get; set; }
        public bool? Complete { get; set; }
        public List<int> Users { get; set; }
    }
}

File: GerenciaMusic360/ClientApp/app/shared/models/marketing-plan.ts
export interface IMarketingPlan {
    id: number;
    marketingId: number;
    taskDocumentDetailId: number;
    position: number;
    notes: string;
    estimatedDateVerification: string;
    required: boolean;
    status: boolean;
    name: string;
    estimatedDateVerificationString: string;
    comments: boolean;
    complete: boolean;
}

export interface IMarketingPlanAutorizes {
    marketingPlanId:  number;
    userVerificationId:  number;
    verificationDate:string;
    checked: boolean;
    verificationDateString: string;
    notes: string;
}